docs/digital-identity.md

```diff
@@ -0,0 +1,201 @@
+
+[the developer docs]: https://docs.publishing.service.gov.uk/manual/govuk-account.html
+
+## Big picture
+
+The `OidcUser` model, which represents a user who has authenticated
+via Digital Identity, holds four special pieces of data:
+
+- The `sub`, or "subject identifier", is a unique identifier assigned
+  by Digital Identity when the account is created, and never changes.
+  Other data, like an email address, can change: so this is what we
+  use to identify users.
+
+- The `legacy_sub` is the user's subject identifier from the old
+  [account manager prototype][].  Only accounts which were created
+  before the Digital Identity migration have this set.  We use this to
+  join up old data.
+
+- The `email` is the user's current email address.  It is set by
+  Digital Identity calling the `/api/oidc-users/:subject_identifier`
+  endpoint.
+
+- The `email_verified` is whether the user's current email address is
+  verified.  It is set by Digital Identity calling the
+  `/api/oidc-users/:subject_identifier` endpoint.  As Digital Identity
+  require immediate verification of email addresses, this is always
+  true.
+
+Communication flows in both directions between account-api and Digital
+Identity:
+
+- When a user is updated or deleted, Digital Identity call the
+  endpoints in the `Internal::OidcUsersController`.
+
+- When a user logs in, we call Digital Identity from the
+  `Internal::AuthenticationController`.
+
+[account manager prototype]: https://github.com/alphagov/govuk-account-manager-prototype/
+
+## Errors
+
+This is a non-exhaustive list of problems that could be due to
+something breaking with the Digital Identity integration.
+
+Firstly, check if the problem is on the Digital Identity side:
+
+- If the user got the problem on `https://account.gov.uk` (or some
+  subdomain of that), then it's Digital Identity.
+- If account-api is persistently unable to connect to
+  `https://oidc.account.gov.uk`, then it's Digital Identity.
+
+If the problem is on the Digital Identity side, or if you need support
+from them, ask for help in `#di-authentication` on Slack.
+
+If the problem is not clearly to do with Digital Identity, read on.
+
+### Email address out of date
+
+Find out from the user what the email address should be and what it's
+currently showing as.
+
+You can find their user record with:
+
+```ruby
+user = OidcUser.find_by(email: "currently showing address")
+```
+
+You can also see if there is a user with the email address they are
+trying to change to:
+
+```ruby
+other_user = OidcUser.find_by(email: "desired new email address")
+```
+
+Digital Identity enforces that email addresses are unique, so we
+shouldn't ever have two users with the same email address.
+
+If there is another user with the desired email address, something may
+have gone wrong with updating *their* user record, which will have to
+be resolved before this user can be updated.
+
+#### Check Sentry for `CapturedSensitiveException` events
+
```

Comment:
I can't find Account API in Sentry, presumably because I don't have access. Should access be granted to the [`#govuk-platform-health` team](https://sentry.io/settings/govuk/teams/govuk-platform-health-team/projects/) like our other GOV.UK apps?

Also could you link to the Sentry project from this section?